schematic/utils/schema_utils.py

```diff
@@ -500,3 +501,20 @@ def get_json_schema_log_file_path(data_model_path: str, source_node: str) -> str
         prefix = prefix_root
     json_schema_log_file_path = f"{prefix}.{source_node}.schema.json"
     return json_schema_log_file_path
+
+
+def export_graph(schema: dict, file_path: str) -> None:
+    """Write object to a pickle file.
+    Args:
+        schema, dict: A data model graph to export
+        file_path, str: File to create
+    """
+    try:
+        with open(file_path, "wb") as file:
+            pickle.dump(schema, file)
+        logger.info(f"The graph was created and saved to '{file_path}'.")
+    except SystemExit as error:
+        logger.exception(
```

Comment:
Also, I think the exception should be raised instead of being "logged". Being "logged" is too silent here. If a path is not valid, it should raise a ValueError. And in the test, you could test ValueError being raised instead of generic Exception

Reply:
done!